### PR TITLE
Add Components

### PR DESCRIPTION
**Pull request by lilyfrancis**
This is a React project, In this section, I added a calculator component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable */
+import React from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
```
Comment (henatan99) [inline src/App.js:7]: - [ ] In this milestone you are requested to make the `App` component a React class-based component. It is the objective's milestone. Kindly refactor it. :pray:

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-useless-constructor */
+/* eslint-disable arrow-body-style */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable-next-line */
+
+import React from 'react';
+
+const ComponentsOfCalculator = () => {
+  return (
+    <div className="container">
```
Comment (henatan99) [inline src/components/Calculator.js:10]: - [ ] In this milestone you are requested to make the `Calculator` component a React class-based component. It is the objective's milestone. Kindly refactor it. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @lilyfrancis , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] In this milestone you are expected to delete boiler plate files in both the public and src folders (images, logos, and icons). :pray: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @lilyfrancis, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**